packages/react-native-passkey-stamper/src/index.ts

```diff
@@ -0,0 +1,207 @@
+import { Passkey } from "react-native-passkey";
+import type { TurnkeyApiTypes } from "@turnkey/http";
+import { getChallengeFromPayload, getRandomChallenge } from "./util";
+
+/**
+ * https://www.w3.org/TR/webauthn-2/#dictionary-credential-descriptor
+ * Copied from https://github.com/f-23/react-native-passkey/blob/17184a1b1f6f3ac61e07aa784c9b64efb28b570e/src/Passkey.tsx#L80C1-L85C2
+ * TODO: can we import this type instead?
+ */
+interface PublicKeyCredentialDescriptor {
+  type: string;
+  id: string;
+  transports?: Array<string>;
+}
+
+/**
+ * Authenticator params expected by the Turnkey API (for authenticator, user, or sub-organization creation)
+ */
+export type TurnkeyAuthenticatorParams =
+  TurnkeyApiTypes["v1AuthenticatorParamsV2"];
+
+/**
+ * Header name for a webauthn stamp
+ */
+const stampHeaderName = "X-Stamp-Webauthn";
+
+export type TPasskeyRegistrationConfig = {
```

Comment:
this is some juicy goodness! 🙌 

another tiiiny nit, and one that's ultimately personal: can we add an extra new line in between each "stanza"/param in this struct? just for readability 😄

Reply:
Yup! You got it 👍